### PR TITLE
Feature 591 fix for csm installation wizard defects

### PR DESCRIPTION
**Pull request by niranjan-n1**
# Description
Addressed the feedback comments and updated unit tests

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| dell/csm#591 |

# Checklist:

- [x] Have you run a grammar and spell checks against your submission?
- [x] Have you tested the changes locally?
- [ ] Have you tested whether the hyperlinks are working properly?
- [ ] Did you add the examples wherever applicable?
- [ ] Have you added high-resolution images?

### CODE REVIEW

File: content/docs/deployment/csminstallationwizard/src/index.html [hunk 1]
```diff
@@ -61,7 +61,7 @@
                     <select id="array" class="form-select" onchange="validateInput(validateForm, CONSTANTS)" required>
                       <option value="" selected>Select the Array</option>
                       <option value="csi-powerstore">PowerStore</option>
-                      <option value="csi-powermax">PowerMax</option>
+                      <option value="powermax">PowerMax</option>
```
Comment (tdawe) [inline content/docs/deployment/csminstallationwizard/src/index.html:64]: Can we use the common "csi-<platform>" name for all drivers or is powermax different for a particular reason?

File: content/docs/deployment/csminstallationwizard/src/csm-versions/csm-1.6.0.properties [hunk 2]
```diff
@@ -1,7 +1,7 @@
 driverVersion=v2.6.0
 vgsnapshotImage=csi-volumegroup-snapshotter:v1.2.0
-replicationImage=dell-csi-replicator:v1.3.0
+replicationImage=dell-csi-replicator:v1.4.0
+podmonImage=podmon:v1.4.0
 authorizationImage=csm-authorization-sidecar:v1.5.0
-migrationImage=dell-csi-migrator:v1.0.0
-powermaxCSIReverseProxyImage=csipowermax-reverseproxy:v2.4.0
```
Comment (chaganti-rajitha) [inline content/docs/deployment/csminstallationwizard/src/csm-versions/csm-1.6.0.properties:6]: Powermax team has updated the charts with latest versions, please change accordingly.

**Review by samihan-dell — APPROVED** [review]

File: content/docs/deployment/csminstallationwizard/src/csm-versions/csm-1.6.0.properties (hunk 2 above)
Comment (niranjan-n1) [reply content/docs/deployment/csminstallationwizard/src/csm-versions/csm-1.6.0.properties:6]: Addressed.

**Review by chaganti-rajitha — APPROVED** [review]

**Review by rsedlock1958 — APPROVED** [review]
I have reviewed this. No edits required.

**Review by tdawe — APPROVED** [review]

**Pull request merged**